### DISCUSSION TITLE
ChatManager.serverSendMessage doesn't have an overload for multiple toPlayers

### DISCUSSION THREAD

**Discussion by Jdance-Media** [SmartlyDressedGames/Unturned-3.x-Community#4023] [Ideas]
Seems weird to invoke the `serverSendMessage` multiple times if I have an exact list of what players need to receive the message.

**Comment by SDGNelson** [comment]
You could make a helper method for calling it with a list of multiple players

**Reply by Jdance-Media** [reply]
Yep. Would require reflection, but easy enough. But still seems like a useful QoL feature for later on.

**Reply by SDGNelson** [reply]
`serverSendMessage` is public (I would advise against using reflection to access internal methods)

**Reply by Jdance-Media** [reply]
My main problem with using `serverSendMessage` multiple times is that it then performs actions that don't need to be performed that many times, and really should only be done once. Currently reflection to access `SendChatEntry` is the only optimized way from my understanding.

It isn't _that_ unoptimized, but it does feels really weird to do so.
